pkg/analytics/track.go

```diff
@@ -359,9 +359,15 @@ func track(event string, success bool, props map[string]interface{}) {
 		return
 	}
 
-	if !okteto.IsContextInitialized() || (!okteto.Context().Analytics && !okteto.IsOktetoCloud()) {
```

Comment:
`okteto.Context().Analytics` is always `false` for vanilla clusters

Reply:
Could we fix that instead, so the behavior is more obvious and consistent?